models/hr/src/index.ts

```diff
@@ -149,8 +158,15 @@ export class TRequest extends TAttachedDoc implements Request {
     tzDueDate!: TzDate
 }
 
+@Model(hr.class.PublicHoliday, core.class.Doc, DOMAIN_HR)
+export class TPublicHoliday extends TDoc implements PublicHoliday {
+  title!: string
+  description!: string
+  date!: TzDate
```

Comment:
I think we should allow to specify few days of public holiday, so it would be good to have date and endDate.

Reply:
Let's make it in next PR.